content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Resource Types
+---
+
+<!-- overview -->
+
+This section provides information on the different resource types you can request, built into Kubernetes.
+
+<!-- body -->
+
+There are 3 different resource types: `cpu`, `memory`, and `storage`.
+
+## CPU 
+The `cpu` resource represents CPU in cores and can be specified as either a decimal (e.g. `1`, `0.5`) or as millicores (e.g. `500m`, `1000m`). `0.5` = `500m`.
```

Comment:
I recommend explaining “CPU” in more detail; see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu for some wording you could migrate to this page.

Reply:
I believe this was resolved below. Feel free to re-add this comment if you feel it wasn't addressed!